File: programmers/python/level00/120825.py
# 문자 반복 출력하기
def solution(my_string, n):
    answer = ''

    for s in my_string:
        answer += s * n

    return answer


# 다른 풀이
# def solution(my_string, n):
#     return ''.join(i * n for i in my_string)

File: programmers/java/level00/120825.java
// 문자 반복 출력하기

import java.util.Arrays;
import java.util.stream.Collectors;

class Solution {
    public String solution(String my_string, int n) {
        return Arrays.stream(my_string.split(""))
            .map(str -> str.repeat(n))
            .collect(Collectors.joining());
    }
}